src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetFinder.java

```diff
@@ -95,7 +95,8 @@ private static FuzzTarget findFuzzTargetByMethodName(Class<?> clazz) {
             "%s must define exactly one of the following two functions:%n"
                 + "public static void fuzzerTestOneInput(byte[] ...)%n"
                 + "public static void fuzzerTestOneInput(FuzzedDataProvider ...)%n"
-                + "Note: Fuzz targets returning boolean are no longer supported; exceptions should be thrown instead of returning true.",
+                + "Note: Fuzz targets returning boolean are no longer supported; exceptions should be thrown instead of returning true.%n"
+                + "Note: When using the @FuzzTest annotation, you will need to set up junit5 or add junit-jupiter-engine version 5 as a dependency.",
```

Comment:
Let's spell it JUnit 5 and refer to `junit-jupiter-engine` via its Maven coordinates `org.junit.jupiter:junit-jupiter-engine` (it's only available in version 5, so we don't have to mention that explicitly).

Reply:
Yes, I have changed the text.